tests/pages/pages.py

```diff
@@ -1153,12 +1153,12 @@ def get(self):
         self.driver.get(self.gov_uk_alerts_url)
 
     def check_alert_is_published(self, page_title, broadcast_content):
```

Comment:
```suggestion
    def check_alert_is_published(self, broadcast_content):
```

Also, is there any way of testing the exception handler works as expected?